iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/compaction/IcebergCompactionUtil.java

```diff
@@ -94,4 +105,48 @@ public static List<DeleteFile> getDeleteFiles(Table table, String partitionPath)
     return Lists.newArrayList(CloseableIterable.transform(filteredDeletesScanTasks,
         t -> ((PositionDeletesScanTask) t).file()));
   }
+
+  /**
+   * Returns target file size as following:
+   * In case of Minor compaction:
+   *  1. When COMPACTION_FILE_SIZE_THRESHOLD is defined, returns it.
```

Comment:
1. When `COMPACTION_FILE_SIZE_THRESHOLD` is supplied via command properties
2. Otherwise, calculates the file size threshold as:
  HiveConf.COMPACTION_FILE_SIZE_THRESHOLD * ...

Reply:
Done